prime-dotnet-webapi/Models/Logging/FrontEndLog.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+
+    [Table("FrontEndLog")]
+    public class FrontEndLog : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Msg { get; set; }
+
+        public string Data { get; set; }
+
+        [Required]
+        public LogType LogType { get; set; }
```

Comment:
It feels a little redundant, but I would actually prefer to see a Date on the log separate from the audit columns using a UTC `DateTime` rather than `DateTimeOffset`.

Reply:
Also seems extra redundant since the data is only accessible through the DB. But can throw in there so down the road we don't have to do a tiny data migration.